src/index.js

```diff
@@ -389,40 +379,11 @@ const stacLayer = async (data, options = {}) => {
       }
     }
 
-    // check if there's only one asset and it's a COG
-    if (addedImagery === false && assetCount === 1 && isAssetCOG(assets[assetKeys[0]])) {
```

Comment:
This block is not really necessary, because it does the same as the block below where just the first COG is used, I think.

Basically, if there's just one asset which is a COG, it will be the same result as if I simply select the first cog available.

This clean-up makes the initialization above (setting assetEntries/Keys/Values/Count) also less complex as we basically only need a list of cogs.

Does this also make sense to you, @DanielJDufour ?

Reply:
yes, great catch!